src/pages/PlayerDetails/PlayerDetails.tsx

```diff
@@ -61,10 +67,53 @@ function PlayerDetails() {
         setDetailType_(type);
     }
 
+    function onAfterLogin() {
+        api.getAccountInfo().then(info => {
+            setAccountInfo(info);
+        })
+    }
+
+    let claimAccountElement = (
+        !wasAlreadyLoggedIn() ? null :
+            selectedPlayer?.name !== accountInfo?.mcName ?
```

Comment:
Is it possible to compare based on the uuid? 
the name may be changed and could result in timing issues